test/environment.rb

```diff
@@ -47,8 +47,9 @@ def self.setup
       File.expand_path('../database.ci.yml', __FILE__)
     end
 
+    use_optimizer
```

Comment:
yea, think this functionality should be automatic performed in `has_ancestry` - either triggered by a parameter or some environmental check.

Reply:
delete all changes to this file